model/riscv_insts_zicsr.sail

```diff
@@ -71,8 +71,8 @@ function clause read_CSR(0b1011100 /* 0xB80 */ @ index : bits(5) if sizeof(xlen)
 
 /* vector */
 function clause read_CSR(0x008) = zero_extend(vstart)
-function clause read_CSR(0x009) = zero_extend(vxsat)
-function clause read_CSR(0x00A) = zero_extend(vxrm)
+function clause read_CSR(0x009) = zero_extend(vcsr[vxsat])
```

Comment:
I think we should probably fix #548 in this MR though. Move all the vector code from this file to `riscv_vext_control.sail` and then delete the duplicates (there are 3 copies for some of them!).

Note the existing code in that file seems to be correct - it includes this fix and also dirties the status flag which the code here doesn't do.

Reply:
Let me know if you can't be bothered though - I can have a go.

Reply:
Sorry for the late reply. Is this okay?

Reply:
Yep LGTM, thanks!